build/tmp/compileJava/source-classes-mapping.txt

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
See all the `<<<<<<<< HEAD` and `>>>>>>>>>` stuff in this file?  Get rid of that - it's part of your Git diff.  Check those code blocks for any redundant code and remove it.